### ISSUE TITLE
[Feature Request] 2018...

### ISSUE THREAD

**Issue by mmattel** [owncloud/core#29998]
This Feature Request is about to wish all contributors a **happy, healthy and successful new year**
:confetti_ball: :rocket: :sparkles:

**Comment by PVince81** [comment]
So we can only close this once the year is finished, fully implemented and qualifies as happy, healthy and successful ? 😉

**Comment by mmattel** [comment]
Maybe we need a PR for autoupdating... 😄 
I see it coming  ...closing in favour of 2019...

**Comment by phil-davis** [comment]
The first 2018 "enhancement" is PR #30047

**Comment by PVince81** [comment]
closing. at least we updated all headers to say 2018 now 😄
